bin/server_docker/docker-entrypoint.sh

```diff
@@ -11,5 +11,6 @@ exec /usr/bin/mysterium_server \
   --config-dir=$OS_DIR_CONFIG \
   --data-dir=$OS_DIR_DATA \
   --runtime-dir=$OS_DIR_RUN \
+  -agreed-terms-and-conditions=$AGREED_TERMS_AND_CONDITIONS \
```

Comment:
We just dropped environmental variables, lets not introduce them again

Reply:
Do you have some alternatives, which would work with Docker and Docker-Compose?

Reply:
We migrated to args style see `e2e/docker-compose.yml`

Reply:
If I understand correctly, declaring arg values in `e2e/docker-compose.yml` is fine, but we need **user input** for terms & conditions - this approach does not work for this.

Is this what you meant? Do you have other suggestions?

Reply:
It's user decision how to given input.
`docker run mysteriumnetwork/mysterium-node:latest --agreed-terms-and-conditions=...`